shell/platform/windows/angle_surface_manager.h

```diff
@@ -32,8 +32,13 @@ class AngleSurfaceManager {
 
   // Creates an EGLSurface wrapper and backing DirectX 11 SwapChain
   // asociated with window, in the appropriate format for display.
-  // Target represents the visual entity to bind to.
-  bool CreateSurface(WindowsRenderTarget* render_target);
+  bool CreateSurface(EGLint width, EGLint height);
+
+  // Resizes backing surface from current size to newly requested size
+  // based on width and height for the specific case when width and height do
+  // not match current surface dimensions.  Returns a value indicating if a
+  // resize was actually performed successfully
```

Comment:
Is there a scenario where we expect to use the return value? Currently the only caller ignores it, so it seems like we should just omit it for now.

Reply:
Fixed in next commit.